mousestyles/dynamics/tests/test_dynamics.py

```diff
@@ -98,49 +98,95 @@ def test_get_prob_matrix_small_interval():
     assert example[0, 2] == 0.2
     assert example[1, 2] == 1.
     assert sum(example[0, :]) == 1.
-
```

Comment:
Actually the below file change is because weiyan create her branch before jarrod merge chengyu's code. After pull from the new master, even we do rebase the branch won't have the new merge content. @harryh1993